### PR TITLE
GitlabCI - Use build.xml as source of truth for project name

### PR DESCRIPTION
**Pull request by symric**

### CODE REVIEW

File: gitlab-ci.sample.yml [hunk 1]
```diff
@@ -190,9 +192,9 @@ Distribute Packages:
   script:
     - pip install awscli
     - export AWS_DEFAULT_REGION="ap-southeast-2"
-    - if [[ $CI_CONFIG_TARBALL_TARGET != "AWS_BUCKET" ]]; then aws s3 cp build/*.tar.gz s3://${CI_CONFIG_TARBALL_TARGET}/${CI_PROJECT_NAME}/ ; fi
+    - if [[ $CI_CONFIG_TARBALL_TARGET != "AWS_BUCKET" ]]; then aws s3 cp build/*.tar.gz s3://${CI_CONFIG_TARBALL_TARGET}/${CI_CONFIG_BUILD_PROJECT}/ ; fi
     - if [[ $CI_CONFIG_TARBALL_TARGET != "AWS_BUCKET" ]]; then aws s3 rm s3://${CI_CONFIG_TARBALL_TARGET}/ --exclude "*" --include '${PROJECT_NAME}/${CI_PROJECT_NAME}-*.${CI_COMMIT_REF_SLUG}.*' --recursive ; fi
```
Comment (nglasl) [inline gitlab-ci.sample.yml]: @symric should this also be changed to use the new variable?

Comment (symric) [reply gitlab-ci.sample.yml]: Yes, copied wrong variable name

Comment (nglasl) [reply gitlab-ci.sample.yml]: Sweet, let me know when that's done and I can approve this for merge :+1:

Comment (nglasl) [reply gitlab-ci.sample.yml]: Can see that change coming through now. Looks good.

**Pull request merged**